packages/api/src/graphql/query.private.ts

```diff
@@ -722,9 +728,9 @@ export const GraphQLQuery = new GraphQLObjectType<undefined, Context>({
 
         after = after ? JSON.parse(base64Decode(after)) : null
 
-        const peers = (await dbAdapter.peer.getPeers()).filter(peer =>
-          peerFilter ? peer.name === peerFilter : true
-        )
+        const peers = (await dbAdapter.peer.getPeers())
+          .filter(peer => (peerFilter ? peer.name === peerFilter : true))
+          .filter(peer => peer.isDisabled !== true)
```

Comment:
Same thing here but with `false` and falsy

```suggestion
          .filter(peer => !peer.isDisabled)
```